osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -13,16 +13,26 @@
 using osu.Game.Beatmaps;
 using osu.Game.Rulesets.Catch.Objects;
 using osu.Game.Rulesets.Catch.Objects.Drawables;
+using osu.Game.Rulesets.Catch.Skinning;
 using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Skinning;
 using osuTK;
 using osuTK.Graphics;
 
 namespace osu.Game.Rulesets.Catch.UI
 {
-    public class Catcher : Container, IKeyBindingHandler<CatchAction>
+    public class Catcher : SkinReloadableDrawable, IKeyBindingHandler<CatchAction>
     {
         public static readonly Color4 DEFAULT_HYPER_DASH_COLOUR = Color4.Red;
 
+        /// <summary>
+        /// The default colour used directly for this <see cref="Catcher"/>'s <see cref="Drawable.Colour"/>.
```

Comment:
this doesn't read well. also please document the above colour if you're going to do so here.

Reply:
Not sure if this is me being dense again but I still kind of don't get what this particular xmldoc is trying to say... In its current state it could be interpreted in a way that makes it misleading (as in, that this colour applies to the whole catcher, which is not *technically* the case - it applies only in hyper-dash state, even if we use `Colour` for that due to additive blending).